attributes/mysql-connectors-community.rb

```diff
@@ -8,8 +8,6 @@
 case node['platform_family']
 when 'rhel'
   case node['platform']
-  when 'amazon'
```

Comment:
It's fine to remove this, but the cookbook will require Chef 13+ after this which should be noted in the readme/metadata